main.c

```diff
@@ -57,6 +57,7 @@ arl_t arl;
 img_t img;
 tns_t tns;
 win_t win;
+const XButtonEvent *cur;
```

Comment:
Think we could afford a more descriptive name. `current_x_event`, `xevent`, `cur_xevent`? "cur" is just so generic.

Also instead of making this a XButtonEvent, make it a XEvent since it could be used for keybindings too.

Reply:
Since only XButtonEvent is being used in the codebase I decided to use that. No strong opinion on it though.